recipes/torsiondrive/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "torsiondrive" %}
+{% set version = "0.8.2" %}
+{% set sha256 = "5fe4a238ffeaf35159bbe9556a37134475aeac0fd40bb69b8f9d538c46f2fc25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - torsiondrive-launch = torsiondrive.launch:main
+    - torsiondrive-api = torsiondrive.td_api:main
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - {{ pin_compatible('numpy', lower_bound='1.11') }}
+    - geometric
+    - python
+
+test:
+  imports:
+    - pytest
+    - torsiondrive
+  requires:
+    - pytest
+  commands:
+    - py.test
+
+about:
+  home: https://github.com/lpwgroup/torsiondrive
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
I do not see the `license_file` in the tarball. The project needs a `MANIFEST.in` in the upstream repo to include it with the source distribution.

Reply:
The `MANIFEST.in` was added. Thanks!